ca/keyreadwriter.go

```diff
@@ -196,9 +248,17 @@ func (k *KeyReadWriter) Write(certBytes, plaintextKeyBytes []byte, kekData *KEKD
 	k.mu.Lock()
 	defer k.mu.Unlock()
 
-	// current assumption is that the cert and key will be in the same directory
-	err := os.MkdirAll(filepath.Dir(k.paths.Key), 0755)
-	if err != nil {
+	// current assumption is that the cert and key will be in the same directory - we
+	// create a temporary directory
```

Comment:
This doesn't look like a temporary directory. Is the comment wrong, or is the path wrong?